snooty/parser.py

```diff
@@ -1230,6 +1231,29 @@ def _locate_text(text: str) -> int:
             if icon_argument:
                 self.validate_and_add_asset(doc, icon_argument, line)
 
+        elif name == "video":
+            sd_req_option_names = [
```

Comment:
[nit] is it possible to get this from the defined options vs defining them twice? im fine with this as is but noting we will need to update two places when needed to be changed

Reply:
It would be cool if a mapping of all directives and their available options existed and is reusable based on `rstspec`, but I'm not sure if that behavior exists or not. I think we'll have to re-define them ourselves for now?